plugins/modules/katello_upload.py

```diff
@@ -94,6 +121,9 @@ def main():
         ),
     )
 
+    if not HAS_RPM:
+        module.fail_json(msg='The rpm Python module is required', exception=RPM_IMP_ERR)
```

Comment:
As we do this lazy import failing anyway, i think we can be even lazyer and move this in the "if yum" section, or inside the `get_rpm_info`. No need to fail file uploads here.

Reply:
`get_rpm_info` has no access to `module`, but I can move that to the `if yum` section, yes.

and need to fix tests. I hate RPM sometimes